protocol/messenger_response.go

```diff
@@ -461,6 +465,14 @@ func (r *MessengerResponse) ActivityCenterNotifications() []*ActivityCenterNotif
 	return ns
 }
 
+func (r *MessengerResponse) SetActivityCenterState(activityCenterState *ActivityCenterState) {
+	r.activityCenterState = activityCenterState
+}
+
+func (r *MessengerResponse) ActivityCenterState() *ActivityCenterState {
```

Comment:
I think it's more idiomatic in Go to not write getters & setters like this, unless they actually perform more computation. Wouldn't it be better to make `activityCenterState` public?

Reply:
I've made it same way as other properties of MessengerResponse, most of them use getters & setters

Reply:
Ouch, okay @MishkaRogachev, let's follow the trend for now.